components/feral-connectd/cdp.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
```

Comment:
Should use [zap]([github.com/uber-go/zap](https://github.com/uber-go/zap)) logger for better logger.